src/js/pages/GranthIndex/raagIndices.js

```diff
@@ -221,7 +221,43 @@ export default {
       },
       {
         name: 'Raag Jaijavanti',
-        pages: [1352, 1353],
+        pages: [1352, 1352],
+      },
+      {
+        name: 'Salok Sehaskriti Mahalla 1',
```

Comment:
Pronunciation/Transliteraion of Mahalla/Mehla is controversial, let's resort to M1/M5? Might have to change Salok M9 as well if we go that route.

@saintsoldierx what do you think?

Reply:
(FYI - the transliteration Mahalla exists in other places on the page which I followed)

Reply:
Yeah that's why I think we might have to change it there as well.